extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -147,13 +147,20 @@ class AmpYoutube extends AMP.BaseElement {
     Services.videoManagerForDoc(this.element).register(this);
   }
 
+  getEmbedUrl() {
+    if (this.getCredentials_() === 'omit') {
```

Comment:
let's refactor a bit to only have the domain part be a variable that's changed and reset of the Url is only written once.